src/ReportGenerator.Core/Reporting/Builders/CoberturaReportBuilder.cs

```diff
@@ -124,16 +145,43 @@ public void CreateClassReport(Class @class, IEnumerable<FileAnalysis> fileAnalys
         /// <param name="summaryResult">The summary result.</param>
         public void CreateSummaryReport(SummaryResult summaryResult)
         {
+            decimal? summaryComplexity = null;
             foreach (var assembly in summaryResult.Assemblies)
             {
+                decimal? assemblyComplexity = null;
                 if (this.packageElementsByName.TryGetValue(assembly.Name, out XElement packageElement))
                 {
                     double packageLineRate = assembly.CoverableLines == 0 ? 1 : assembly.CoveredLines / (double)assembly.CoverableLines;
                     double packageBranchRate = assembly.TotalBranches.GetValueOrDefault() == 0 ? 1 : assembly.CoveredBranches.GetValueOrDefault() / (double)assembly.TotalBranches.GetValueOrDefault();
 
                     packageElement.Add(new XAttribute("line-rate", packageLineRate.ToString(CultureInfo.InvariantCulture)));
                     packageElement.Add(new XAttribute("branch-rate", packageBranchRate.ToString(CultureInfo.InvariantCulture)));
-                    packageElement.Add(new XAttribute("complexity", "NaN"));
+
+                    foreach (var @class in assembly.Classes)
```

Comment:
Didn't find a better way to do that to assembly level complexity. Do you have any suggestion?